README.md

```diff
@@ -25,9 +25,11 @@ scd("source_data_scd", {
   // A field that stores a timestamp or date of when the row was last changed.
   timestamp: "updated_at",
   // The source table to build slowly changing dimensions from.
-  source: {
-    schema: "dataform_scd_example",
-    name: "source_data",
+    hash: "hash_value", // OPTIONAL
+    // A field that stores the hash value of the fields that we want to track changes in. If you do not want to use the hash comparison, you may omit this field or set it to null
```

Comment:
I think the comments in this file are above the property, not below - so you want to swap these, and move them above `// The source table to build...` line

Reply:
Right, fixed!